tron/core/jobrun.py

```diff
@@ -231,6 +245,7 @@ def handle_action_run_state_change(self, action_run: ActionRun, event):
 
         # propagate all state changes (from action runs) up to state serializer
         self.notify(self.NOTIFY_STATE_CHANGED)
```

Comment:
Would it be easier to put this in the `notify` method? (the log is a type of notification?)

Reply:
We won't have access to the action name at that point.